### ISSUE TITLE
promisifyAll(redis.Multi.prototype) garbles node http client

### ISSUE THREAD

**Issue by mnezh** [charlieduong94/util-promisifyAll#4]
Looks like `promisifyAll` applied to `redis.Multi.prototype` garbles node http client
```javascript
const https = require('https')
const promisifyAll = require('util-promisifyall')
const redis = require('redis')
promisifyAll(redis.Multi.prototype) // that thing breaks it all

https.get({
  protocol: 'https:',
  host: 'google.com'
}, (res) => {
  res.setEncoding('utf8')
  res.on('data', () => {})
  res.on('end', () => {
    // here you can see a lot headers added to HTTP stream, like below:
    // GET / HTTP/1.1
    // Host: google.com
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // undefined: undefined
    // Connection: close
    //
    // without promisifyAll(redis.Multi.prototype) is looks like:
    // GET / HTTP/1.1
    // Host: google.com
    // Connection: close
    console.log(res.req._header);
  })
}
).on('error', (err) => {
  console.log(err)
})
```

**Comment by charlieduong94** [comment]
Hello @mnezh, thanks for reporting this! I'll look into finding a fix sometime this weekend.

**Comment by charlieduong94** [comment]
@mnezh I just published version `1.0.3` which contains the fix for this. Please let me know if you see any more issues!
